scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/remotes/postgres/PostgresResultsCellRepository.java

```diff
@@ -44,12 +47,43 @@ public ResultsProtocol.OwnerRole allocate(final SpecificationId specificationId)
     }
   }
 
+  @Override
+  public Optional<ResultsProtocol.OwnerRole> claim(final SpecificationId specificationId)
+  {
+    try (
+        final var connection = this.dataSource.getConnection();
+        final var claimSimulationAction = new ClaimRequestAction(connection)
+    ) {
+      claimSimulationAction.apply(specificationId.id());
+      logger.info("Claimed scheduling request with specification id {}", specificationId);
```

Comment:
please either move this log until after we know the claim is successful or change it to "Attempted to claim..."

Reply:
Ahh true, there are a few lines below this that could cause the whole claim to fail. I've moved this to right before the final `return` in this method.

Thanks!